var/spack/repos/builtin/packages/py-snakemake-executor-plugin-googlebatch/package.py

```diff
@@ -17,6 +17,11 @@ class PySnakemakeExecutorPluginGooglebatch(PythonPackage):
 
     license("MIT")
 
+    version("0.5.0", sha256="242ddb2348db1837a0676e991b257e0924791d3dab20aa8f89d63c548adfd1cd")
```

Comment:
Version `@0.5.0` adds following constraint

```python
snakemake = "^8.18.0"
```

The constraint is in dev dependencies, but this plugin needs Snakemake, so it might be better to add it.